### ISSUE TITLE
Please help, i got this error while playing like always

### ISSUE THREAD

**Issue by Samu49** [Monika-After-Story/MonikaModDev#10072]
I'm sorry, but an uncaught exception occurred.

While running game code:

TypeError: 'NoneType' object is not iterable

-- Full Traceback-

Full traceback:

File "C:\Users\Samuel\Desktop\Doki Doki Literature Club\renpy\bootstrap.py", line 295, in bootstrap renpy.main.main()

File "C:\Users\Samuel\Desktop\Doki Doki Literature Club\renpy\main.py", line 490, in main renpy.persistent.update(True)

File "C:\Users\Samuel\Desktop\Doki Doki Literature Club\renpy\persistent.py", line 373, in update merge(other)

File "C:\Users\Samuel\Desktop\Doki Doki Literature Club\renpy\persistent.py", line 319, in merge val merge_func(old, new, pval)

File "C:\Users\Samuel\Desktop\Doki Doki Literature Club\renpy\persistent.py", line 271, in dictset_merge current.update(old)

TypeError: 'NoneType' object is not iterable

Windows-8-6.2.9200

Ren'Py 6.99.12.4.2187 Monika After Story 0.12.14

**Comment by Booplicate** [comment]
Did this happen mid-game? Doesn't look like it.

**Comment by Samu49** [comment]
> Did this happen mid-game? Doesn't look like it.

Yes it did happened while playing but didn't happened again for now so maybe it's just random

**Comment by Booplicate** [comment]
Looks like it's on renpy side, I don't think we can do much here, but I will keep this open until we migrate to a newer version.

**Comment by Samu49** [comment]
> Looks like it's on renpy side, I don't think we can do much here, but I will keep this open until we migrate to a newer version.

Alright, thanks for the help
